cformers/interface.py

```diff
@@ -65,11 +62,14 @@ def get_modes(self):
     'bigscience/bloom-7b1': ModelUrlMap(
         cpp_model_name="bloom",
         int4_fixed_zero="https://huggingface.co/ayushk4/bigscience-.-bloom-7b1/resolve/main/int4_fixed_zero.bin"),
+    'gpt2': ModelUrlMap(
+        cpp_model_name="gpt2",
+        int4_fixed_zero="https://huggingface.co/kamalojasv/gpt2_int4_fixes_zero/resolve/main/model-quant.bin"),
```

Comment:
Can you stick to `int4_fixed_zero.bin` name of the file and `gpt2` as name of the huggingface model repo?

Reply:
This should be trivial to edit on the hugginface-interface.

Reply:
Sure

Reply:
Done